### ISSUE TITLE
Doesn't Seem to learn or overfit even one image

### ISSUE THREAD

**Issue by I-CANT-CODE** [jvanvugt/pytorch-unet#11]
I am attempting to train the UNET but it doesn't seem to be learning and just spits out garbage values. I even tried to overfit it to a single example but it doesn't even learn that. Ill attach a pdf of the notebook I am using to train.
[GIT.pdf](https://github.com/jvanvugt/pytorch-unet/files/4206809/GIT.pdf)

**Comment by jvanvugt** [comment]
Hmm at first sight I can't see any bugs in your code. I haven't trained the model myself in a while, but there is some open source work that uses it. Maybe you could use it to compare to your own code?  
https://github.com/lyft/nuscenes-devkit/blob/master/notebooks/Reference%20Model.ipynb
http://www.andrewjanowczyk.com/pytorch-unet-for-digital-pathology-segmentation/

**Comment by 0xzayd** [comment]
It's the last layer. Add  Sigmoid if you have binary classifciation or Softmax if you have multiple classes
